### PR TITLE
[RPA-1645] Create dependabot.yml and add rush update workaround

### PR DESCRIPTION
**Pull request by star-in-the-jar**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

---

I agree to the terms of the RunBotics Contributor License Agreement.

### CODE REVIEW

**Review by marcin-depcik — APPROVED** [review]

File: .github/workflows/update-pnpm-lock.yml [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        shell: bash
+        run: |
+          git push origin ${{ github.ref_name }}
+
+      - name: Clean up
+        run: |
+          git status
+          git add .
+          git status
+          git reset --hard
```
Comment (Aeri4a) [inline .github/workflows/update-pnpm-lock.yml:86]: Why status -> add -> status and then reset? Is it for displaying it in logs?

Comment (Aeri4a) [reply .github/workflows/update-pnpm-lock.yml:86]: Didn't you add .npmrc by accident?

Comment (star-in-the-jar) [reply .github/workflows/update-pnpm-lock.yml:86]: yup, git status is  for logs.
no, .npmrc is added for rush to be able to install RunBotics own packages for which to get you need authentication token

Comment (star-in-the-jar) [reply .github/workflows/update-pnpm-lock.yml:86]: This 'Clean up' action is needed to remove all files tracked by git to be able to run rb version

**Review by star-in-the-jar — COMMENTED** [review]
self review

File: .github/workflows/update-pnpm-lock.yml (hunk 1 above)
Comment (Aeri4a) [reply .github/workflows/update-pnpm-lock.yml:86]: Cool, thanks

**Review by Aeri4a — APPROVED** [review]

**Pull request merged**